docs-website/adoptionStoriesIndexes.json

```diff
@@ -89,6 +89,42 @@
       "category": "B2B & B2C",
       "description": "<i>“We looked around for data catalog tool, and DataHub was a clear winner.”</i> <br /> <br /> Zynga levels up data management using DataHub, highlighting its role in enhancing data management, tracing data lineage, and ensuring data quality."
     },
+    {
+      "name": "Miro",
+      "slug": "miro",
+      "imageUrl": "/img/logos/companies/miro.png",
+      "imageSize": "medium",
+      "link": "https://miro.com/careers/life-at-miro/tech/data-products-reliability-the-power-of-metadata/",
+      "linkType": "blog",
+      "tagline": "Data Products Reliability: The Power of Metadata",
+      "category": "B2B & B2C",
+      "platform": "cloud",
+      "description": "<i>\"Leveraging our Datahub catalog, we have centralized metadata access for all data products. This integration eliminates the dependency on Airflow metadata alone for defining contracts, enabling flexible definitions for both building blocks and business metrics.\""
```

Comment:
note that if we make this a `.jsx` file, we can use proper react markup for the descriptions instead of embedding it in a string

looks like we're not closing the <i> tag here